Formula/deno.rb

```diff
@@ -0,0 +1,35 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno.git",
+      :tag      => "v0.2.5",
+      :revision => "320577c131c8058ba0065f26119ce1ce18bc23d2"
+
+  bottle do
+  end
+
+  depends_on "ninja" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build
+  depends_on "rust" => :build
+
+  def install
+    rm_rf ["node_modules", "buildtools", "build_overrides", "testing", "tools/clang"]
+    system "python", "tools/setup.py"
+    system "python", "tools/build.py", "-C", "target/release"
+    ENV["DENO_BUILD_MODE"] = "release"
+    # system "python", "tools/test.py"
```

Comment:
This should be removed if not needed.

Reply:
It would be good to run this after building the binary, at the moment it fails unfortunately.

Reply:
Happy to delete this (and the ENV), but would like to add them back in the future (once the test passes).

Reply:
Happy to delete this (and the ENV), but would like to add them back in the future (once the test passes).